README.md

```diff
@@ -176,6 +176,18 @@ If you want to override all the values with the new one you can configure `multi
 
 An example of this is `INFRA-CONTAINER` where the tag `container.state` will always display the last state (`running`, `stopped`, etc) instead of a list of all the states the entity has gone through.
 
+You can also configure the tag name that you want for the entity tags we synthesize from your telemetry attributes. 
```

Comment:
The first sentence seems mostly redundant with the second. Would it make sense to remove it?

Reply:
True, I'll reword it